src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -728,6 +728,8 @@ namespace Microsoft.Management.Deployment
         String LogOutputPath;
         /// Continues the install even if the hash in the catalog does not match the linked installer.
         Boolean AllowHashMismatch;
+        /// Continues the install even if the malware scan detects potential malware.
+        Boolean IgnoreMalwareScan;

```

Comment:
Please revert all the changes under Microsoft.Management.Deployment

First, this only applies to installation using -m <manifest>, this code will never be invoked from Com entrypoint. It is only applicable to local manifest testing. Second, even if this code will be invoked, this is bypassing the group policy.

Reply:
Reverted